graphix/pattern.py

```diff
@@ -1416,12 +1416,46 @@ def run_pattern(self, backend, **kwargs):
         result = exe.run()
         return result
 
-    def perform_pauli_measurements(self, leave_input=False, use_rustworkx=False):
+    def perform_pauli_measurements(self, leave_input=False, use_rustworkx=False, ignore_pauli_with_deps=False) -> None:
```

Comment:
```suggestion
    def perform_pauli_measurements(
        self, leave_input: bool = False, use_rustworkx: bool = False, ignore_pauli_with_deps=False
    ) -> None:
```

Reply:
`ruff format` enforces the single-line version, which is expected because the line is only 119 characters long, falling within the 120-character limit.

Reply:
I don't care code formats as long as `: bool` is added and CI passes.

Reply:
Fixed in df843d9. Sorry, I didn't notice your suggestion to add type annotations. You added them to `leave_input` and `use_rustworkx` (parameters not modified in this PR) but did not add them to `ignore_pauli_with_deps`, which is the new parameter introduced in this PR. I have added the missing type annotations to all the arguments.

Reply:
Oh I overlooked that. Thanks!